dpnp/dpnp_iface_indexing.py

```diff
@@ -198,3 +199,42 @@ def nonzero(a):
         return dpnp_nonzero(a)
 
     return call_origin(numpy.nonzero, a)
+
+
+def tril_indices(n, k=0, m=None):
+    """
+    Return the indices for the lower-triangle of an (n, m) array.
+
+    Parameters
+    ----------
+    n : int
+        The row dimension of the arrays for which the returned
+        indices will be valid.
+
+    k : int, optional
+        Diagonal offset (see `tril` for details).
+
+    m : int, optional
+        The column dimension of the arrays for which the returned
+        arrays will be valid.
+        By default `m` is taken equal to `n`.
+
+    Returns
+    -------
+    inds : tuple of arrays
+        The indices for the triangle. The returned tuple contains two arrays,
+        each with the indices along one dimension of the array.
+
+    """
+
+    if not use_origin_backend():
+        if not isinstance(n, int):
+            raise TypeError("n must be an integer")
```

Comment:
Could you please change this checks a bit.
It is better to reuse fallback for any exception or error handling.
Example of this might be here https://github.com/IntelPython/dpnp/blob/836fe4cd601cb07e785c29a18f3f3f3f1eb54634/dpnp/fft/dpnp_iface_fft.py#L100-L109